src/examples/server/simpleStreamableHttp.ts

```diff
@@ -174,6 +174,19 @@ app.post('/mcp', async (req: Request, res: Response) => {
   }
 });
 
+// Handle GET requests for SSE streams (now using built-in support from StreamableHTTP)
```

Comment:
The "now" will age badly:
```suggestion
// Handle GET requests for SSE streams (using built-in support from StreamableHTTP)
```